sdk/purview/purview-workflow-rest/samples-dev/workflowRunSample.ts

```diff
@@ -23,7 +23,7 @@ dotenv.config();
  */
 async function workflowRunsList(
   client: PurviewWorkflowClient,
-  queryParameter: ListWorkflowRunsParameters
+  queryParameter: WorkflowRunsListParameters
```

Comment:
Why this file's changes didn't reflect in `samples` folder? I would suggest you to run `npx dev-tool samples publish -f ` again.

Reply:
Thanks, I've fixed this.